<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bottom</class>
 <widget class="QWidget" name="Bottom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bottom</string>
  </property>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>0</y>
     <width>20</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="enableDriverPushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>模块使能</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>3</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模块类型&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>TYPE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFlash">
   <property name="geometry">
    <rect>
     <x>1065</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;写入Flash&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>FLASH</string>
   </property>
  </widget>
  <widget class="QLabel" name="devLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>20</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>3</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DEV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="workModePushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>25</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工作模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="helpLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;帮助信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Online Help</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>0</y>
     <width>3</width>
     <height>40</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>985</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOAD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectedPushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>25</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CONNECTED</string>
   </property>
  </widget>
  <widget class="QLabel" name="ratioLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>3</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模块减速比&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>RATIO</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>170</width>
     <height>3</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ifErrorPushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>25</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NO ERROR</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbID">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>0</y>
     <width>65</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;切换模块&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="firmLabel">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>0</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>3</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;固件版本号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>FIRM </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="helpTextEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>535</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>7</horstretch>
     <verstretch>4</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;帮助信息显示栏&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdateID">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>更新ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enableDriverPushButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>25</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ENABLE</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>2</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
